tests/cli/test_cli_run.py

```diff
@@ -0,0 +1,48 @@
+import pytest
+from click.testing import CliRunner
+
+from dbt.cli.main import cli
+from tests.cli.fixtures import BaseConfigProject
+
+
+class TestCli(BaseConfigProject):
```

Comment:
Intention is to have this best the basic happy path of click entry point while more complex cases are being tested by functional tests? Love that!

Nit: If that's the case, can we add some comment in this file to make it a bit clear?